openmc/tracks.py

```diff
@@ -266,6 +266,55 @@ def plot(self):
             track.plot(ax)
         return ax
 
+    def write_tracks_to_vtk(self, filename):
+        """Creates a VTK file of the tracks
+
+        Parameters
+        ----------
+        filename : str
+            Name of the VTK file to write.
+
+        Returns
+        -------
+        vtk.vtkStructuredGrid
+            the VTK object
```

Comment:
This needs to be updated -- I am assuming we'd want to return the `vtkPolyData` object? Also, right now, the method is returning the filename which should also be changed.

Reply:
Yep good idea, I've updated the doc string and am now returning the vtkpolydata